docs/source/mayavi/installation.rst

```diff
@@ -295,78 +326,39 @@ The bleeding edge: Git
 
 If you want to get the latest development version of Mayavi (e.g. for
 developing Mayavi or contributing to the documentation), we
-recommend that you check it out from github: Mayavi is hosted on github,
-with the rest of the Enthought open source packages: the 'ETS' (Enthought
-Tool Suite): https://github.com/enthought
+recommend that you check it out from github: Mayavi is hosted on github
+with its own repository: https://github.com/enthought/mayavi as part of the
+Enthought open source packages (ETS): https://github.com/enthought/ets
 
 Mayavi depends on several packages that are part of ETS.  It is highly
 likely that the in-development mayavi version may depend on some feature
-of an as yet unreleased component.  Therefore, it is very convenient to
-get all the relevant ETS projects that Mayavi recursively depends on in
-one single checkout. For this purpose a script `ets.py` is available.
-
- #. Make sure there is no other ETS package installed in your pythonpath::
-
-     $ python
-     >>> import enthought
-     Traceback (most recent call last):
-       File "<stdin>", line 1, in <module>
-     ImportError: No module named enthought
-
-    If you *don't* get the ImportError (e.g. importing ``enthought`` succeeds),
-    then there is no way to install the git Mayavi version over it (even if you
-    put it first in your PYTHONPATH), because the older (setuptools managed)
-    ETS packages will get picked up too and they will mess up things. This
-    behavior might be surprising if you are new to setuptools.
-
-    So for example if you use Ubuntu or Debian, you need to first remove all ETS 
-    packages (in Ubuntu 9.04, you need to remove all of these: ``mayavi2 python-apptools
-    python-enthoughtbase python-envisagecore python-envisageplugins
-    python-traits python-traitsbackendwx python-traitsgui``).
-
- #. Create an empty directory and download in it the `ets.py` script from
-    `https://github.com/enthought/ets/raw/master/ets.py <https://github.com/enthought/ets/raw/master/ets.py>`_
-
- #. To get just the sources for mayavi and all its dependencies do this::
-
-      $ python ets.py clone
+of an unreleased component.  It is very convenient to get all the
+relevant ETS projects that Mayavi recursively depends on with a single
+checkout.  The ETS repository provides an `ets` module, which allows
+pulling all other ETS projects from github, and other useful features.
 
-    This will download from github the source code for the entire ETS.
+To install ETS::
 
-    .. note:: 
+  mkdir ets
+  cd ets
+  git clone http://github.com/enthought/ets.git
+  cd ets
+  python setup.py develop
+  cd ..
+  ets -h
 
-       The `ets.py` downloads the entire ETS, which is more than you need
-       to build Mayavi. As the extra packages have additional
-       dependencies, they may render the build harder. You can remove
-       safely the following directories::
+To clone all the ETS projects::
```

Comment:
@kitchoi I agree with your qualm / temptation -- suggest replacing this with a reference to https://github.com/enthought/ets